### PR TITLE
Ako/ add vercel DR action

### PR DESCRIPTION
**Pull request by ali-hosseini-deriv**

### CODE REVIEW

File: .github/actions/vercel_DR_publish/action.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  using: composite
+  steps:
+    - name: Install Vercel CLI
+      run: npm install -g vercel
+      shell: bash
+
+    - name: Pull Vercel project
+      shell: bash
+      run: |
+        vercel pull --yes --token=${{ inputs.VERCEL_TOKEN }} --scope=${{ inputs.VERCEL_SCOPE }}
```
Comment (balakrishna-deriv) [inline .github/actions/vercel_DR_publish/action.yml:33]: we should use inputs directly as per the security team's recommendation. Can we set these in env before we use it?

Comment (ali-hosseini-deriv) [reply .github/actions/vercel_DR_publish/action.yml]: this is done

File: .github/actions/vercel_DR_publish/action.yml [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+      run: npm install -g vercel
+      shell: bash
+
+    - name: Pull Vercel project
+      shell: bash
+      env:
+        VERCEL_TOKEN: ${{ inputs.VERCEL_TOKEN }}
+        VERCEL_SCOPE: ${{ inputs.VERCEL_SCOPE }}
+      run: |
+        vercel pull --yes --token=${{ env.VERCEL_TOKEN }} --scope=${{ env.VERCEL_SCOPE }}
```
Comment (balakrishna-deriv) [inline .github/actions/vercel_DR_publish/action.yml:36]:
```suggestion
        vercel pull --yes --token=$VERCEL_TOKEN --scope=VERCEL_SCOPE
```

Comment (balakrishna-deriv) [reply .github/actions/vercel_DR_publish/action.yml:36]: using env. directly is still prone to injection

**Review by balakrishna-deriv — APPROVED** [review]

**Pull request merged**